src/components/AboutPage.js

```diff
@@ -1,15 +1,140 @@
 import React from 'react';
+import '../styling/about.css';
+import { Button, Col, Container, Row } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+import { Link } from 'react-router-dom';
+{/* TODO
+  - images
+  - style cleanup
+  - spacing cleanup
+*/}
 
 const AboutPage = () => (
-  <div class="container mt-5">
-    <div class="row">
-      <div class="col"></div>
-      <div class="col-8">
-        <h1>About us</h1>
-      </div>
-      <div class="col"></div>
-    </div>
-  </div>
+  <Container fluid="md">
+
+    <Row>
```

Comment:
could you add some padding at the top so it looks even (currently being eaten up by navbar)

Reply:
ya was fiddling around with spacing but haven't pushed yet